core/api/src/main/java/org/eclipse/sensinact/core/model/SensinactModelManager.java

```diff
@@ -24,11 +24,17 @@ public interface SensinactModelManager extends CommandScoped {
 
     ModelBuilder createModel(String model);
 
+    ModelBuilder createModel(String modelPackageUri, String model);
+
     Model getModel(String model);
 
+    Model getModel(String modelPackageUri, String model);
+
     void deleteModel(String model);
 
-    void registerModel(String model);
```

Comment:
Is this breaking change necessary? It seems like it might be, so I'm not asking you to put it back.

Reply:
I think we should talk about the register and delete Model API anyway.  Especially as we don't have any implementation yet.